source/documentation/incidents/process.html.md.erb

```diff
@@ -9,57 +9,46 @@ review_in: 4 weeks
 This document describes our incident handling process (inspired by work elsewhere in MOJ like the Cloud Platform).
 
 ## Confirm that an event constitutes an incident
-------
 
-we define an incident as an event which:
+We define an incident as an event which:
 
-* An event that requires immediate response to return normal service
+* Requires immediate response to return normal service
 * Severely degrades user experience of the service
-* Compromise of application security, resulting in a breach or potential breach
-* Potential for loss or compromise of data
+* Compromises application security, resulting in a breach or potential breach
+* Has the potential for loss or compromise of data
 
 Examples:
 
 * Live service is unreachable due to denial of service
-* Connectivity to a database is down and so services cannot be accessed.
-* A problem in one service means another is no longer functioning i.e. api gateway in Sirius causing issues in Use An LPA
+* Connectivity to a database is down and so services cannot be accessed
+* A problem in one service means another is no longer functioning e.g. api gateway in Sirius causing issues in Use An LPA
 * Misconfiguration means that secrets are exposed on a live service
 * Unauthorized access to an API which can result in attackers accessing data
 
 ## Declare an incident
-------
 
 From your teams channel you can use the OPG incidents slack tool to declare an incident.
 
 `/opg-incident Something's happened`
 
-You'll be prompted to fill in extra information in a GUI like this:
+Slack will pop open a form to to fill in extra information. At this stage, you only need to select whether its a live incident or something that's already happened that you'd like to report. All the other information can be filled in later.
 
-{{ }}
+The slack bot will automatically post a message into the `#opg-incident` channel and create an entry on [the incident response site][incident-response-site]. **This marks the offical start of an incident**
 
-At this stage, just select if its a live incident or something you'd just like to report. All the other information can be filled in later.
+### Immediate first steps
 
-The slack bot will automatically post a message into the `#opg-incidents` channel. **This marks the offical start of an incident**
+As soon as the tool has posted a message into `#opg-incident` you should do these steps immediatly
 
-
-### immediate first steps
-
-As soon as the tool has posted a message into `#opg-incidents` you should do these steps immediatly
-
-1. Hit the create communtications channel button in the message (only available for active incidents).
-2. Page the on-call incident lead.
-
-These steps should be done
+1. Use the create communtications channel button in the message (only available for active incidents)
+2. Page the on-call incident lead
+3. If appropriate, start a huddle on the Slack channel for further discussion
```

Comment:
Personal pref, but huddles are awkward for anything that also needs slack messages. Generally we've gone with meet calls for incidents.

Reply:
On the flip side I've found Huddles quite good because of how they're integrated into the Slack channel. But I've generalised to "conference call" because the technology doesn't matter and we might not have access to Meet in the future.

Reply:
Yeah tech agnostic is the way to go!